src/main/webapp/resources/js/pages/analysis/components/AnalysisDetails.jsx

```diff
@@ -0,0 +1,254 @@
+/*
+ * This file renders the details for the analysis as well as,
+ * lazily loads the Samples, Share, and Delete components (component
+ * is only loaded when the corresponding tab is clicked
+ */
+
+/*
+ * The following import statements makes available all the elements
+ *required by the components encompassed within
+ */
+
+import React, { useContext, Suspense, lazy, useEffect } from "react";
+import {
+  Button,
+  Checkbox,
+  Input,
+  List,
+  Col,
+  Row,
+  Tabs,
+  Select,
+  Spin,
+  Divider,
+  Typography
+} from "antd";
+
+import { AnalysisContext, isAdmin } from "../../../contexts/AnalysisContext";
+import { AnalysisDetailsContext } from "../../../contexts/AnalysisDetailsContext";
+import { getI18N } from "../../../utilities/i18n-utilties";
+import { showNotification } from "../../../modules/notifications";
+import { SPACE_MD } from "../../../styles/spacing";
+
+import { getVariablesForDetails } from "../../../apis/analysis/analysis";
+
+import {
+  formatDate,
+  getHumanizedDuration
+} from "../../../utilities/date-utilities";
+
+const AnalysisSamples = React.lazy(() => import("./AnalysisSamples"));
+const AnalysisShare = React.lazy(() => import("./AnalysisShare"));
+const AnalysisDelete = React.lazy(() => import("./AnalysisDelete"));
+const Option = Select;
```

Comment:
`Options` is unused

Reply:
You have deconstructed the `Select` object here to be able to use the `Option` on it's own later, but later you used the non-deconstructed form `Select.Option`.  `Option` by itself **is** unused.

Reply:
Fixed in 9c4e640d5c452bf1455f9b93a1b1499a418283e6